moped-editor/src/components/GridTable/Search.js

```diff
@@ -34,14 +38,11 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   advancedSearchRoot: {
-    width: "calc(100% - 32px)",
-    zIndex: "3",
-    paddingLeft: "16px",
-    paddingRight: "16px",
-    [theme.breakpoints.down("md")]: {
-      paddingLeft: "23px",
-      paddingRight: "14px",
+    width: `calc(100% - ${theme.spacing(6)})`,
+    [theme.breakpoints.down("sm")]: {
+      width: `calc(100% - ${theme.spacing(4)})`,
```

Comment:
this makes the filter dropdown width track the same width changes that the search box above

Reply:
thaaaaaank you